tests/release/pipelines/fbc_release.go

```diff
@@ -261,27 +239,29 @@ func assertReleasePipelineRunSucceeded(devFw, managedFw framework.Framework, dev
 		if !releasePr.IsDone() {
 			return fmt.Errorf("release pipelinerun %s in namespace %s did not finish yet", releasePr.Name, releasePr.Namespace)
 		}
-		GinkgoWriter.Println("Release PR: ", releasePr.Name)
```

Comment:
Why change this? We have the releasePr or else line 233 would have failed. The name is nice to see

Reply:
Restored it.